src/lib/consensus/proof_of_stake.ml

```diff
@@ -738,24 +743,24 @@ module Make_str (A : Wire_types.Concrete) = struct
                 ledger staker_addr )
         in
         let%bind () =
-          [%with_label_ "Account is for the default token"] (fun () ->
+          with_label "Account is for the default token" (fun () ->
```

Comment:
Revert. We agreed to get rid of `snarkydef`, not the whole of snarky's ppx.

Reply:
oh, I thought that's what we meant